tests/CMakeLists.txt

```diff
@@ -967,3 +967,9 @@ add_legacy_tests(
   igraph_sample_dirichlet
   igraph_sample_sphere
 )
+
+# tessellation
+add_legacy_tests(
```

Comment:
is adding a new `add_legacy_tests` Ok?

Reply:
Yes, just change the heading to "lattices".